### PR TITLE
Pre 3.3 fixes

### PR DESCRIPTION
**Pull request by poshul**
## Description

@timosachsenberg and I discussed yesterday, and think we are ready to do a 3.3 release with our new fixes and features. @cbielow @t0mdavid-m @jpfeuffer can you double check that I've added the relevant bits to the changelog?

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

___

### **PR Type**
Documentation, Enhancement

___

### **Description**
- Updated the `CHANGELOG` to include new features, enhancements, and bug fixes for the upcoming OpenMS 3.3.0 release.
- Added Hendrik Weisser back to the `ACTIVE_MAINTAINERS` list.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>CHANGELOG</strong><dd><code>Update CHANGELOG with new features and fixes for OpenMS 3.3.0</code></dd></summary>
<hr>

CHANGELOG

<li>Added new features and enhancements for OpenMS 3.3.0.<br> <li> Documented changes in compatibility and new functionalities.<br> <li> Listed various bug fixes and improvements.<br>

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS/pull/7664/files#diff-ecec88c33adb7591ee6aa88e29b62ad52ef443611cba5e0f0ecac9b5725afdba">+20/-0</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr><tr><td><strong>Miscellaneous</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>ACTIVE_MAINTAINERS</strong><dd><code>Update ACTIVE_MAINTAINERS to include Hendrik Weisser</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tools/ACTIVE_MAINTAINERS

- Re-added Hendrik Weisser to the list of maintainers.

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS/pull/7664/files#diff-a5741757ad431e36b3ccfc5d0383c004eca973aea46aa409d4bb0fffcc24178e">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by t0mdavid-m** [comment]
lgtm!

File: CHANGELOG [hunk 1]
```diff
@@ -17,6 +17,10 @@ PR - Pull Request (on GitHub), i.e. integration of a new feature or bugfix
 What's new:
 - Changes breaking backwards compatibility: 
   - the QualityControl TOPP tool has some renamed parameters and supports output directories (#7497)
+- Fast peak extraction from MSExperiments (#7628)
```
Comment (timosachsenberg) [inline CHANGELOG]:
```suggestion
- Low-level  peak extraction helper for MSExperiments (#7628)
```

**Review by timosachsenberg — APPROVED** [review]
I think it is good to go.
For next iteration we could think about generating changelog entries automatically

File: CHANGELOG [hunk 2]
```diff
@@ -17,6 +17,10 @@ PR - Pull Request (on GitHub), i.e. integration of a new feature or bugfix
...
+- Now use Numpy 2 (#7539)
```
Comment (cbielow) [inline CHANGELOG:21]:
```suggestion
- pyOpenMS: use Numpy2 (#7539)
```

**Review by cbielow — COMMENTED** [review]
looks good

**Pull request merged**